pandas/conftest.py

```diff
@@ -31,6 +32,16 @@ def pytest_addoption(parser):
                      help="Fail if a test is skipped for missing data file.")
 
 
+def pytest_collection_modifyitems(items):
```

Comment:
I don't think we'll want this permanently. This is just to track down #22675 in case it happens on this branch.

Reply:
Do we need this as part of the ultimate change or no?

Reply:
I'm tempted to leave it in master for a week or two to see if we catch any of the failures in action. I've made myself a reminder to remove it.

Reply:
sighhhhhhhhhhhhhhhhhhhhhhhhhhhh. we got one: https://travis-ci.org/pandas-dev/pandas/jobs/429657921#L2280, but that doesn't actually fail the right test, because the warning is in the file object's `__del__`, just prints out to stderr rather than raising an exception (to fail the test).

I'm going to do some debugging on the branch.